internal/configure/hosts/hosts.go

```diff
@@ -206,12 +206,12 @@ func ParseHosts(data string) ([]*HostConfig, error) {
 			return nil, err
 		}
 
-		if _, ok := exist[hc.GetHost()]; ok {
-			return nil, errno.ERR_DUPLICATE_HOST.
-				F("duplicate host: %s", hc.GetHost())
+		if _, ok := exist[hc.GetName()]; ok {
+			return nil, errno.ERR_DUPLICATE_NAME.
+				F("duplicate host: %s", hc.GetName())
```

Comment:
ditto

Reply:
I guess what you want to propose here is the modification of the error message in errno
I think this error message should refer to the duplication of host names, so the error message should be modified here to ERR_DUPLICATE_NAME
If there is an error in my understanding, please remind me and I will modify the code according to your guidance as soon as possible or modify it back to the previous version.